File: FrondEnd/Model/Usuario.py
from pydantic import BaseModel,EmailStr

class UsuarioCreate(BaseModel):
    Nombre_Usuario: str
    Nombre_Completo: str
    Correo: EmailStr
    Contraseña: str
    Rol: str

class UsuarioBase(BaseModel):
    Usuario_ID: int
    Nombre_Usuario:str
    Nombre_Completo:str
    Correo:EmailStr
    Contraseña:str
    Rol:str

    class Config:
        orm_mode = True

File: FrondEnd/interno/Models/usuario.go
package Models

type Usuario struct {
	Nombre_Usuario  string `json:"Nombre_Usuario"`
	Nombre_Completo string `json:"Nombre_Completo"`
	Correo          string `json:"Correo"`
	Contraseña      string `json:"Contraseña"`
	Rol             string `json:"Rol"`
}
